requirements.txt

```diff
@@ -37,3 +37,4 @@ wandb>=0.16.4
 xdg-base-dirs>=6.0.1
 psutil>=6.0.0
 huggingface_hub[hf_transfer]>=0.1.8
+yq>=3.0.0
```

Comment:
The [python package `yq`](https://github.com/kislyuk/yq) is not a real yaml tool. It is just a wrapper around `jq` ("jq wrapper for YAML/XML/TOML documents") and thus it does not handle the [`line` operator](https://mikefarah.gitbook.io/yq/operators/line) that we need. We need to use the proper `yq` tool, https://github.com/mikefarah/yq, which is not a python package.

Reply:
> The [python package `yq`](https://github.com/kislyuk/yq) is not a real yaml tool. It is just a wrapper around `jq` ("jq wrapper for YAML/XML/TOML documents") and thus it does not handle the [`line` operator](https://mikefarah.gitbook.io/yq/operators/line) that we need. We need to use the proper `yq` tool, https://github.com/mikefarah/yq, which is not a python package.

Ok if that's the case so indeed my change doesn't make sense. 
I am trying to think of any way of preventing users running into similar problem, maybe some point in README.md to ask user install /usr/bin/yq, and not use Python yq wrapper?

Reply:
A readme mention in a useful place could be helpful. We only "need" yq to report the line number of any schema violations. So yq is not a hard requirement but a soft requirement to improve error messages when the qna.yaml is invalid.